app/src/main/java/de/westnordost/streetcomplete/quests/barrier_bicycle_barrier_type/AddBicycleBarrierType.kt

```diff
@@ -0,0 +1,31 @@
+package de.westnordost.streetcomplete.quests.barrier_bicycle_barrier_type
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.edits.update_tags.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement.BICYCLIST
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement.BLIND
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement.CAR
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement.PEDESTRIAN
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement.WHEELCHAIR
+
+class AddBicycleBarrierType : OsmFilterQuestType<BicycleBarrierType>() {
+
+    override val elementFilter = """
+        nodes with barrier=cycle_barrier and !cycle_barrier
+    """
+    override val commitMessage = "Add specific cycle barrier type"
+    override val wikiLink = "Key:cycle_barrier"
+    override val icon = R.drawable.ic_quest_barrier // TODO
+    override val isDeleteElementEnabled = true
+
+    override val questTypeAchievements = listOf(CAR, PEDESTRIAN, BLIND, WHEELCHAIR, BICYCLIST)
```

Comment:
How about just bicyclist? These are bicycle barriers after all.

Reply:
car, pedestrian seems pointless and I will remove it

but it is also useful for wheelchair/blind users (tripple chicane is annoying also for people on wheelchair and I suspect that it could be useful info for blind)

Reply:
ok